### ISSUE TITLE
Some taxonomies are created multiple times in the database

### ISSUE THREAD

**Issue by sielay** [getlackey/lackey-cms#69]
# Some taxonomies are created multiple times in the database

## Type

 * Bug

## Environment

 * Version / Commit hash 0.8.1

## Expected behaviour

```sql
select name, "taxonomyTypeId", count(*) from taxonomy group by name, "taxonomyTypeId"
```

counts should be equal 1

## Actual behaviour

```sql
select name, "taxonomyTypeId", count(*) from taxonomy group by name, "taxonomyTypeId"
```

counts are higher than 1

**Comment by sielay** [comment]
```sql

delete from taxonomy where id in (

select t.id from taxonomy t, (select * from (select name, "taxonomyTypeId", count(*) from taxonomy group by name, "taxonomyTypeId") as foo where count > 1) as boo
where boo.name = t.name and t."taxonomyTypeId" = boo."taxonomyTypeId" and t.id not in (select "taxonomyId" from "userToTaxonomy"
union all
select "taxonomyId" from "roleToTaxonomy"
union all
select "taxonomyId" from "contentToTaxonomy"
union all
select "taxonomyId" from "templateToTaxonomy"))

```

**Comment by sielay** [comment]
Fixed with #70
